cross/libimagequant/Makefile

```diff
@@ -0,0 +1,23 @@
+PKG_NAME = libimagequant
+PKG_VERS = 2.12.6
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/ImageOptim/libimagequant/archive
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
```

Comment:
Please set `PKG_DIST_FILE` to avoid "version-only" file in distrib/ and digest

Reply:
done.